app/uploaders/supporting_evidence_attachment_uploader.rb

```diff
@@ -2,11 +2,12 @@
 
 # Files uploaded as part of a form526 submission that will be sent to EVSS upon form submission.
 class SupportingEvidenceAttachmentUploader < CarrierWave::Uploader::Base
-  include ValidateFileSize
   include SetAwsConfig
   include ValidatePdf
 
-  MAX_FILE_SIZE = 50.megabytes
+  def size_range
+    1...50.megabytes
```

Comment:
do we want this as `1..50.megabytes` (two dots) to include 50mb?

maybe even ` 1..50.1.megabytes.round`? --seems like most file managers report to the nearest 10th of a megabyte.

Reply:
this would inclued 50 mb https://github.com/carrierwaveuploader/carrierwave/blob/f15f4ba71dd2c78e8df494bf65938bbaf13960f9/lib/carrierwave/uploader/file_size.rb#L35

Reply:
```
irb(main):001:0> (1...50).max
=> 49
```
triple dots won't include the top of the range